.github/workflows/manubot.yaml

```diff
@@ -0,0 +1,40 @@
+name: Manubot
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    name: Build Manuscript
```

Comment:
Maybe `Build and Deploy Manuscript` to be more accurate, and to avoid duplicate name of the `Build Manuscript` step below.

Reply:
FWIW you can also omit this name, and I think it will show up in the `actions` tab as just the name of the whole config file.

Reply:
FWIW I think you can omit this and in the `Actions` tab it will just show up as the config file name at the top.